frontend/js/components/map.js

```diff
@@ -139,15 +138,11 @@ export class LeafletMap extends React.Component {
             return <IssueMarker {...props} />
         })
 
-        let extents = geojsonExtent(JSON.parse(JSON.stringify(geojson)));
```

Comment:
Yes, you are right.
But there is still some difference in how the map locations are handled. When I have this change in the js code and click a marker, all markers vanish and then re-appear (I guess after loading all issues again, but I'll have to debug to be sure)